config/config.go

```diff
@@ -236,6 +236,29 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 				ogc.APIURL += "/"
 			}
 		}
+		for _, wcc := range rcv.WechatConfigs {
+			wcc.ApiURL = c.Global.WeChatAPIURL
+			if wcc.ApiURL == "" {
```

Comment:
can u rename `ApiURL` to `APIURL`? Keep the same pattern found on other provider configs.